### DISCUSSION TITLE
How can I get the client ip instead of strongswan's internal ip

### DISCUSSION THREAD

**Discussion by ferdiina** [strongswan/strongswan#1661] [Q&A]
hello all
i already create site to site ipsec vpn .  strongswan 5.7.2  Centos7
leftsubnet 192.168.1.0/24   
pfsense ip 192.168.1.254  and with internet IP  
rightsubnet 192.168.5.0/24
strongswan ip 192.168.5.254  and with internet IP  
when 192.168.5.0/24 access 192.168.1.0/24 all traffic source ip is 192.168.5.254

conf:

> 
> conn %default

>             keylife=3600s
>             keyingtries=%forever
>             authby=psk
>             keyexchange=ikev2
>             reauth=no
>             ikelifetime=86400s
>             dpddelay=30s
>             dpdtimeout=120s
>             dpdaction=restart
>             ike=aes128-sha1-modp1024
>             esp=aes128-sha1-modp1024
>             lifetime=86400s
>             rekeymargin=540s
>             type=tunnel
>             compress=no
>             authby=secret
>             keyingtries=%forever
>             mobike=no
>             fragmentation=yes
>             auto=start

> conn 1-5
```
>         left=y.y.y.y
>         leftid="left"
>         right=x.x.x.x
>         rightid="right"
>         leftsubnet=192.168.1.0/24
>         rightsubnet=192.168.5.0/24
>         auto=start
```

missing some configation ?  i try this "nat_traversal=no" but it's not avalibe in strongswan 5.7.2

**Comment by tobiasbrunner** [comment]
There might be a NAT from which you have to [exclude VPN traffic](https://docs.strongswan.org/docs/5.9/howtos/forwarding.html#_general_nat_problems). Otherwise, see #196.
